app/models/annotation/AnnotationUploadService.scala

```diff
@@ -53,8 +54,7 @@ class AnnotationUploadService @Inject()(tempFileService: TempFileService, nmlPar
       nmlParser.parse(
         name,
         inputStream,
-        sharedParsingParameters.overwritingDatasetId,
-        sharedParsingParameters.isTaskUpload,
```

Comment:
all members of sharedParsingParameters are passed to `nmlParser.parse` anyway. So just pass the whole `sharedParsingParameters` object